libs/ledger-live-common/src/families/ton/utils.ts

```diff
@@ -0,0 +1,2 @@
+// Encapsulate for LLD et LLM
```

Comment:
If this file is not used by Desktop nor by Mobile, you can remove it.

Reply:
fixed on commit `5fe562009d6943af46996bdbf77630ae08db22c4`